taxcalc/tests/test_pufcsv.py

```diff
@@ -80,19 +80,21 @@ def test_agg(tests_path, puf_path):  # pylint: disable=redefined-outer-name
         raise ValueError(msg)
     # create aggregate diagnostic table using sub sample of records
     fullsample = pd.read_csv(puf_path)
-    rn_seed = 80  # to ensure two-percent sub-sample is always the same
-    subsample = fullsample.sample(frac=0.02,  # pylint: disable=no-member
+    rn_seed = 80  # to ensure sub-sample is always the same
+    subfrac = 0.02  # sub-sample fraction
+    subsample = fullsample.sample(frac=subfrac,  # pylint: disable=no-member
                                   random_state=rn_seed)
     rec_subsample = Records(data=subsample)
     calc_subsample = Calculator(policy=Policy(), records=rec_subsample)
     adt_subsample = multiyear_diagnostic_table(calc_subsample, num_years=nyrs)
     # compare combined tax liability from full and sub samples for each year
     taxes_subsample = adt_subsample.loc["Combined Liability ($b)"]
-    reltol = 0.01  # maximum allowed relative difference in tax liability
+    reltol = 0.04  # maximum allowed relative difference in tax liability
```

Comment:
@andersonfrailey, @hdoupe,

@martinholmer flagged this line change for me on the phone today. 

The normal differences between the 2% sample and the full sample rose sharply when we introduced the new file. This could be symptomatic of a problem, and it is certainly intriguing. Weights may be a source of interest -- Martin noticed that they are not used to define the 2% sample. 

Could you investigate when you want a break from your other work?